src/main/java/org/broadinstitute/hellbender/utils/fermi/FermiLiteAssembly.java

```diff
@@ -17,6 +18,25 @@ public FermiLiteAssembly( final List<Contig> contigs ) {
     public Contig getContig( final int idx ) { return contigs.get(idx); }
     public List<Contig> getContigs() { return contigs; }
 
+    public int computeN50() {
+        final int nContigs = contigs.size();
+        if ( nContigs < 1 ) return 0;
+        if ( nContigs == 1 ) return contigs.get(0).getSequence().length;
+
+        final int[] lengths = new int[nContigs];
+        for ( int idx = 0; idx != nContigs; ++idx ) {
+            lengths[idx] = contigs.get(idx).getSequence().length;
+        }
+        Arrays.sort(lengths);
+        final int totalSize = Arrays.stream(lengths).sum();
```

Comment:
I'm not experienced enough in Java to know for sure (or in this project to know if it matters / how this function will be used), but would it be more performant to add up totalSize in the for loop that gets the lengths?

Reply:
Yes, that would probably be faster.  I'll change it.